public/javascripts/game_partials/_5_modal.js

```diff
@@ -148,15 +149,16 @@ $(document).on('hint-indicator-change', function(event, count) {
 });
 
 $(document).on('game-over', function(event, how) {
-
+  // TODO: Dont we need a time taken here too?
```

Comment:
there is no need to send time here.
If time-out gets triggered that implies that time is 120 seconds.